src/form/wr.jl

```diff
@@ -126,29 +126,6 @@ function constraint_voltage_magnitude_bounds_soft(pm::_PM.AbstractWRModel, n::In
     JuMP.@constraint(pm.model, w >= vm_min^2 - w_vio)
 end
 
-"`p[arc_from]^2 + q[arc_from]^2 <= w[f_bus]/tm*ccm[i]`"
-function _PM.constraint_power_magnitude_sqr_on_off(pm::_PM.AbstractQCWRModel, n::Int, i, f_bus, arc_from, tm)
-    w    = _PM.var(pm, n, :w, f_bus)
-    p_fr = _PM.var(pm, n, :p, arc_from)
-    q_fr = _PM.var(pm, n, :q, arc_from)
-    ccm  = _PM.var(pm, n, :ccm, i)
-    z    = _PM.var(pm, n, :z_branch, i)
-
-    # TODO see if there is a way to leverage relaxation_complex_product_on_off here
-    w_lb, w_ub = InfrastructureModels.variable_domain(w)
-    ccm_lb, ccm_ub = InfrastructureModels.variable_domain(ccm)
-    if isinteger(z)
-        z_lb = z_ub = z
-    else
-        z_lb, z_ub = InfrastructureModels.variable_domain(z)
-    end
```

Comment:
Just noting here, while I remember, that this "if block" is the main difference between this and the PowerModels implementation. If this function is necessary for anything, we should (a) document the purpose of this change and (b) find a way to implement it without redefining the PowerModels method.

Reply:
This goes back 5 years, but if I remember right, the intent is that only damaged lines have a binary variable.  Non-damaged lines have the non-switching power flow constraints, while damaged lines have the switching power flow constraints.